src/Microsoft.ML.Data/Data/SchemaDefinition.cs

```diff
@@ -349,57 +340,90 @@ public static SchemaDefinition Create(Type userType, Direction direction = Direc
                 ((direction & Direction.Write) == Direction.Write && (x.CanWrite && x.GetSetMethod() != null))) &&
                 x.GetIndexParameters().Length == 0);
 
-            var memberInfos = (fieldInfos as IEnumerable<MemberInfo>).Concat(propertyInfos).ToArray();
+            return (fieldInfos as IEnumerable<MemberInfo>).Concat(propertyInfos).ToArray();
+        }
 
-            foreach (var memberInfo in memberInfos)
+        public static bool NeedToCheckMemberInfo(MemberInfo memberInfo)
```

Comment:
Please, change the access modifier to `internal` for this method and GetNameAndCustomAttributes() for the reasons I had explained here:
https://github.com/dotnet/machinelearning/pull/5114#discussion_r431450626

Reply:
Sorry, I forget this one